Source/ACE.DatLoader/DatReader.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System;
+using System.IO;
+
+namespace ACE.DatLoader
+{
+    public class DatReader
+    {
+        public int Offset { get; set; }
+        public byte[] buffer { get; private set; }

```

Comment:
Stylecop rules dictate that we should use uppercase for the first letter of this name: ex. `Buffer`